extensions/compressx/src/compressVideo.tsx

```diff
@@ -0,0 +1,25 @@
+import { showToast, Toast, getSelectedFinderItems } from "@raycast/api";
+import { exec } from "child_process";
+
+export default async function main() {
+  let filePaths: string[];
+
```

Comment:
Maybe you can check if it's installed? Extensions like [Eagle](https://www.raycast.com/Yukai/eagle) does that

Reply:
Update with installation check. Can you help to review again? Thanks